parking/backend/db/dbaccess.py

```diff
@@ -74,3 +81,26 @@ class DbAccess(object):
                 logger.warning("Tried to allocate user : '{}' when they already had an allocation.".format(user_id))
                 return False
         return True
+
+    async def get_parking_lot_allocations(self, park_id: int) -> List[Record]:
+        async with self.pool.acquire() as conn:
+            records: List[Record] = await conn.fetch(c.ALLOCATIONS_SELECT, park_id)
+        return records
+
+    async def delete_allocation(self, user_id: str) -> None:
```

Comment:
Again, no indication of whether this actually works or not. How are you validating this stuff?

Reply:
There are actual tests in tests/test_db.py?

Reply:
And what if something goes wrong that is outside the scope of the unit test?